workflow/controller/controller_test.go

```diff
@@ -953,6 +953,7 @@ func TestNotifySemaphoreConfigUpdate(t *testing.T) {
 	for i := 0; i < 3; i++ {
 		key, _ := controller.wfQueue.Get()
 		controller.wfQueue.Done(key)
+		controller.wfQueue.Forget(key)
```

Comment:
Why this change?

Reply:
It was a bug in the previous test I believe, if the rate limiter has any entries, it will be added later. This is makes the test flakey as a result because it now relies on timings.

Reply:
:+1: